### ISSUE TITLE
Remove files not used by app.

### ISSUE THREAD

**Issue by dromazmj** [driverta/drive_ionic#20]
The repo contains alot of files that are not used by the app.  We need to go through and look at every file and decide if it is needed or not.

Care should be taken to make sure each file deleted is not used by ionic in any means.  For example the .sourcemap files are questionable.

**Comment by driverta** [comment]
I agree. I will clean up the extra files before our big weekend migrating the database.
